test/trento_web/auth/jwt_auth_plug_test.exs

```diff
@@ -83,15 +68,6 @@ defmodule TrentoWeb.JWTAuthPlugTest do
     test "should return an error is the refresh token signature is valid but it's expired", %{
       conn: conn
     } do
-      expect(
```

Comment:
Is not the expiration test based on this value?
Simply commenting because I see two different mocked values for the time through the tests

Reply:
as long as `expired  > current` and `non-expired < current`  the test is ok